orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -138,7 +138,9 @@ def __repr__(self) -> str:
         return result
 
     def __eq__(self, other):
-        if not isinstance(other, _TensorStub):
+        if other is None:
```

Comment:
I think line 145 already checks this case.

Reply:
Oops, I missed that, but check order is important. It must be checked before the `not isinstance(other, _TensorStub)` branch otherwise an unexpected exception is raised